comp/process/runner/runner_test.go

```diff
@@ -25,8 +25,13 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
 )
 
+type testDeps struct {
+	fx.In
+	Runner Component
+}
```

Comment:
I think using only `Component` should be enough

Reply:
Oops! missed one :) thanks